doc/flutter-helpers.md

```diff
@@ -124,7 +124,7 @@ By default a resource is disposed by calling its `.dispose` method. However if t
 
 The resource name is threaded (as per `->`) through the `:dispose` form. Most of the time it will be simply a method or a function.
 
-Last, you can introduce intermediate values to use in resource initialization via `:let`:
+Lastly, you can introduce intermediate values to use in resource initialization via `:let`:
```

Comment:
Not really a typo, I think both are valid, no?

Reply:
I appreciate you taking a look!

I am not an expert, but I think 'lastly' sounds more natural here. [Collins](https://grammar.collinsdictionary.com/english-usage/what-is-the-difference-between-last-and-lastly) seems to agree:

> Lastly has a much more common use. You use it to introduce a final point in a discussion, ask a final question, give a final instruction, or mention a final item in a list.

However, not very significant and perhaps not a typo as you stated, so please let me know if you want this bit reverted.